hack/update-codegen.sh

```diff
@@ -28,6 +28,8 @@ fi
 
 source $(dirname $0)/../vendor/knative.dev/test-infra/scripts/library.sh
 
+boilerplate="${REPO_ROOT_DIR}/hack/boilerplate/boilerplate.go.txt"
```

Comment:
```suggestion
local boilerplate="${REPO_ROOT_DIR}/hack/boilerplate/boilerplate.go.txt"
```
?

Reply:
Doesn't that only work in functions?

Reply:
We're not in a function - is `local` applicable here?

Reply:
one big function :)